src/config/wallets.ts

```diff
@@ -178,6 +190,16 @@ export const supportEvmWalletObj = {
     isSupportMobileApp: true,
     ethExtension: 'ethereum',
   },
+  [SupportWallet.HanaEvm]: {
+    img: require('/src/assets/img/logo-hana.png'),
+    name: 'Hana Wallet (EVM)',
+    source: SupportWallet.HanaEvm,
+    walletUrl: 'https://hanawallet.io/',
+    guideUrl: 'https://hanawallet.io/',
+    isSupportBrowserExtension: true,
+    isSupportMobileApp: false,
+    ethExtension: 'ethereum',
```

Comment:
`Provider` looks like `hanaWallet.ethereum` (`window.ethereum` connects to MetaMask).
![image](https://user-images.githubusercontent.com/92044428/200243361-2840e8d8-0c4c-46b7-a9e0-642f79609aee.png)